### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by pradeepnschrodinger**
Let's update the [issue template](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates) as per #489.

Note that this repo has had a single issue template, which'll be used whenever user tries to open up a new issue. 
This is the legacy workflow, and it's defined in [.github/ISSUE_TEMPLATE.md](https://github.com/schrodinger/fixed-data-table-2/blob/master/.github/ISSUE_TEMPLATE.md)

The newer workflow, however, is to specify multiple templates and let the user choose whatever seems best.
eg (seen in [react-virtualized](https://github.com/bvaughn/react-virtualized/issues/new/choose)):
image

For that, instead of modifying the existing `ISSUE_TEMPLATE.md` file, we create a `.md` file for each of the templates. This PR will be for having something like the "Bug Reports" template.

The only change within the contents are the codesandbox link. I specified a slightly [modified version](https://codesandbox.io/s/fixed-data-table-example-3gd9x) from the one mentioned in #489.

Ofc, we don't have to do all these right away. The main goal from #489 was to just update the codesandbox link. But since we're already touching the issue template, it might be best to discuss these now.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue-template.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+name: Issue Template
```
Comment (pradeepnschrodinger) [inline .github/ISSUE_TEMPLATE/issue-template.md:2]: We could rename this to `Bug Reports`, if it's clearer to the user. eg: 
image

File: .github/ISSUE_TEMPLATE/issue-template.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+about: Template for issues
+title: ''
```
Comment (pradeepnschrodinger) [inline .github/ISSUE_TEMPLATE/issue-template.md:4]: Not sure if we wanna specify title here. Maybe something like "[BUG]" ?

File: .github/ISSUE_TEMPLATE/issue-template.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or - -->
+<!--- Fork the CodePen to reproduce https://codesandbox.io/s/fixed-data-table-example-3gd9x ->
```
Comment (pradeepnschrodinger) [inline .github/ISSUE_TEMPLATE/issue-template.md:26]: Updated code sandbox link.

nit: change `CodePen` to `CodeSandBox`

Comment (pradeepnschrodinger) [inline .github/ISSUE_TEMPLATE/issue-template.md:26]: nit: change `CodePen` to `CodeSandBox`

File: .github/ISSUE_TEMPLATE/issue-template.md (hunk 1 above)
Comment (wcjordan) [reply .github/ISSUE_TEMPLATE/issue-template.md:2]: Updating to Bug Reports sounds good to me

File: .github/ISSUE_TEMPLATE/issue-template.md (hunk 3 above)
Comment (wcjordan) [reply .github/ISSUE_TEMPLATE/issue-template.md:26]: I don't have a problem with making that change.  Maybe we could have both a simple and an advanced sandbox example?

**Review by wcjordan — APPROVED** [review]
lgtm

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -14,7 +23,9 @@
 
 ## Steps to Reproduce (for bugs)
 <!--- Provide a link to a live example, or - -->
-<!--- Fork the CodePen to reproduce https://codepen.io/alphalpha/pen/oqKJgG ->
+<!--- Fork any of these code sandbox templates: ->
+<!---   *  Basic template: https://codesandbox.io/s/fixed-data-table-basic-example-cwds0 ->
+<!---   *  Advanced template: https://codesandbox.io/s/fixed-data-table-advanced-example-3gd9x ->
```
Comment (pradeepnschrodinger) [inline .github/ISSUE_TEMPLATE/bug-report.md:28]: Let's have a [basic](https://codesandbox.io/s/fixed-data-table-basic-example-cwds0) and an [advanced](https://codesandbox.io/s/fixed-data-table-advanced-example-3gd9x) example for the user's convenience.

**Review by pradeepnschrodinger — COMMENTED** [review]
@wcjordan , I'll wait for another LGTM for the new changes

**Pull request merged**